package.json

```
@@ -2,14 +2,16 @@
   "name": "function",
   "version": "1.0.0",
   "description": "aqa task",
-  "main": "./cypress/integration/cypressTests.js",
+  "main": "./cypress/integration/testingWebTables.js",
+  "type": "module",
   "scripts": {
     "test": ""
```

add here a few commands how to run the test

```suggestion
    "test:debug": "cypress open",
    "test:headed":"cypress run"
```

And after that u can run the tests using `npm run test:debug` or `npm run test:headed` in your terminal.